Cargo.toml

```diff
@@ -21,7 +21,6 @@ tokio = { version = "1.0", features = ["macros", "rt-multi-thread"] }
 warp = "0.3"
 rlp = "0.4.5"
 web3 = "0.13.0"
-eip-712 = "0.1.1"
 serde_json = "1.0.57"
 ethabi = "12.0.0"
```

Comment:
We can probably remove ethabi too right?

Reply:
We could probably remove a decent amount of these, looking at automated solutions now.

Reply:
It's used in error handling currently. That trait implementation could likely be pruned but I might make a separate PR for this?